src/libraries/System.Diagnostics.PerformanceCounter/Directory.Build.props

```diff
@@ -3,7 +3,8 @@
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
     <IsWindowsSpecific>true</IsWindowsSpecific>
-    <PackageVersion>5.0.1</PackageVersion>
+    <ServicingVersion>1</ServicingVersion>
+    <!-- This should be pinned to 5.0.0.1 going forward as it is part of aspnetcore ref pack.-->
```

Comment:
Instead of doing this on every assembly that is part of the aspnetcore ref pack can we have a list of assemblies that are part of it and pin the version for all those in a common location?

Reply:
Ah you did it this way cause not all of them are pinned to the same version